uber/templates/preregistration/preregbase.html

```diff
@@ -69,7 +71,7 @@
                 extra: 0
             }]
         };
```

Comment:
Yup, that's fine. Do we want to change this to check the `gets_free_shirt` property instead of checking for a staff badge?